### DISCUSSION TITLE
Temporal Upscaler option not appearing (No DLSS) in REFramewrok menu RE4

### DISCUSSION THREAD

**Discussion by Crapmonkey90** [praydog/REFramework#1149] [General Questions]
![image](https://github.com/user-attachments/assets/3937431f-c38e-4923-bd74-e9e133a8019b)

It doesn't show up, other options load. Not sure why I can't adjust DLSS or even tell if its working

**Comment by praydog** [comment]
You are supposed to get the `pd-upscaler` builds from this page: https://github.com/praydog/REFramework/actions

You have a release build, not an upscaler build. You can get the latest upscaler build here for RE4: https://github.com/praydog/REFramework/actions/runs/11770014195/artifacts/2169163401

**Reply by Crapmonkey90** [reply]
Thank you so much for the help. Didn't realize DLSS functionality wasn't in the main release.

**Reply by loopqq** [reply]
> Si suppone che si ottengano le build da questa pagina: https://github.com/praydog/REFramework/actions`pd-upscaler`
> 
> Hai una build di rilascio, non una build di upscaler. Puoi ottenere l'ultima build di upscaler qui per RE4: https://github.com/praydog/REFramework/actions/runs/11770014195/artifacts/2169163401

It doesn't appear to me in Resident Evil 7 Do you know why? I tried to download all the frameworks for RE7 but nothing

**Reply by ZerOneX** [reply]
I am having a hard time here to "enable" the path tracing on my RE7 game, can someone enlighten me please? I got the release build RE7 REF Nightly 01286, what do I have to do next? Thank you all in advance

I got the RE7.zip from the Upscaler branch, but even though I do not have the DLSS option.... some crucial information is missing

I’ve ensured that all the required files are present in the same folder as well.

-PDPerfPlugin.dll
-ffx_fsr3_x64.dll and ffx_fsr3upscaler_x64.dll (for FSR)
-nvngx_dlss.dll (for DLSS)

**Comment by lawnmowerdrangelo-gif** [comment]
When i launh the game with upscaler release, it comes back to steam after few seconds after launch. Non upscaling mod work normal, anyone succesful?
